content/docs/other-guides/virtual-dev/getting-started-minikube.md

```diff
@@ -209,7 +209,7 @@ $ minikube start --cpus 4 --memory 8096 --disk-size=40g
 ### Install Kubeflow on an existing Kubernetes cluster
 
 Now that you have a Kubernetes cluster running - the Minikube cluster - follow the
-[existing Kubernetes cluster]((/docs/started/k8s/overview/)) instructions for installing
+[existing Kubernetes cluster]((/docs/started/k8s/)) instructions for installing
```

Comment:
Thanks! The original link should be fine - it's the double parentheses that are causing the problem.

Reply:
PR https://github.com/kubeflow/website/pull/1088 has fixed the link. Please would you revert this change?

Reply:
Please revert this change, and it doesn't fix the broken link. There's a merge conflict now, because another PR has fixed the link. If reverting the change doesn't resolve the conflict, you can sync the PR with your upstream remote to pick up the fixed link. If things get into too much of a tangle, consider closing this PR and creating a new one with just the fix on line 28.